doc/man/man7/elektra-bootstrapping.7

```diff
@@ -1,12 +1,12 @@
 .\" generated with Ronn-NG/v0.10.1
 .\" http://github.com/apjanke/ronn-ng/tree/0.10.1.pre1
-.TH "ELEKTRA\-BOOTSTRAPPING" "7" "November 2020" ""
+.TH "ELEKTRA\-BOOTSTRAPPING" "7" "November 2021" ""
 .SH "NAME"
 \fBelektra\-bootstrapping\fR \- default backend
 .P
 One important aspect of a configuration library is the out\-of\-the\-box experience\. How does the system work before anything is configured? The optimal situation is that everything fully works, and applications, that just want to load and store configuration, do not see any difference between out\-of\-the\-box behavior and a well\-configured, fine\-tuned system\.
 .P
-To support that experience, a so\-called \fBdefault backend\fR is responsible in the case that nothing was configured so far\. It must have a storage that is able to store full Elektra semantics\. To avoid reimplementation of storage plugins, for default storage plugins (\fBstorage\fR or in code \fBKDB_STORAGE\fR) is used\. A resolver plugin (\fBresolver\fR or in code \fBKDB_RESOLVER\fR) takes care of the inevitable portability issues\. The \fBdefault backend\fR stores configuration in \fBKDB_DB_FILE\fR\. One can easily avoid the usage of the default backend by simple mounting another backend to \fB/\fR\.
+To support that experience, a so\-called \fBdefault backend\fR is responsible in the case that nothing was configured so far\. It must have a storage that is able to store full Elektra semantics\. To avoid reimplementation of storage plugins, a default storage plugin (\fBstorage\fR or in code \fBKDB_STORAGE\fR) is used\. A resolver plugin (\fBresolver\fR or in code \fBKDB_RESOLVER\fR) takes care of the inevitable portability issues\. The \fBdefault backend\fR stores configuration in \fBKDB_DB_FILE\fR\. One can easily avoid the usage of the default backend by simple mounting another backend to \fB/\fR\.
```

Comment:
```suggestion
To support that experience, a so\-called \fBdefault backend\fR is responsible in the case that nothing was configured so far\. It must have a storage that is able to store full Elektra semantics\. To avoid reimplementation of storage plugins, a default storage plugin (\fBstorage\fR or in code \fBKDB_STORAGE\fR) is used\. A resolver plugin (\fBresolver\fR or in code \fBKDB_RESOLVER\fR) takes care of the inevitable portability issues\. The \fBdefault backend\fR stores configuration in \fBKDB_DB_FILE\fR\. One can easily avoid the usage of the default backend by simply mounting another backend to \fB/\fR\.
```

Excellent work, keep it up :+1: